src/test/java/easytests/core/mappers/QuestionsMapperTest.java

```diff
@@ -1,111 +1,107 @@
 import org.mockito.ArgumentCaptor;
-import org.mockito.Mockito;
 import static org.mockito.Mockito.times;
 import static org.mockito.Mockito.verify;
 import org.springframework.beans.factory.annotation.Autowired;
 
+
 /**
- * @author firkhraag
+ * @author RisaMagpie
  */
 public class QuestionsMapperTest extends AbstractMapperTest {
 
+    protected QuestionsSupport questionsSupport = new QuestionsSupport();
+
     @Autowired
     private QuestionsMapper questionsMapper;
 
     @Test
     public void testFindAll() throws Exception {
-        final List<QuestionEntity> questionsEntities = this.questionsMapper.findAll();
-        Assert.assertEquals((long) 3, (long) questionsEntities.size());
+        final List<QuestionEntity> questionsFoundedEntities = this.questionsMapper.findAll();
+        Assert.assertEquals( 3,  questionsFoundedEntities.size());
+
+        Integer index = 0;
+        for (QuestionEntity questionEntity: questionsFoundedEntities) {
+            final QuestionEntity questionFixtureEntity = this.questionsSupport.getEntityFixtureMock(index);
+
+            this.questionsSupport.assertEquals(questionFixtureEntity, questionEntity);
+            index++;
+        }
     }
 
     @Test
     public void testFind() throws Exception {
-        final QuestionEntity question = this.questionsMapper.find(1);
-        Assert.assertEquals((long) 1, (long) question.getId());
-        Assert.assertEquals("test1", question.getText());
-        Assert.assertEquals((long) 1, (long) question.getQuestionTypeId());
-        Assert.assertEquals((long) 1, (long) question.getTopicId());
+        final QuestionEntity questionFixtureEntity = this.questionsSupport.getEntityFixtureMock(0);
+
+        final QuestionEntity questionFoundedEntity = this.questionsMapper.find(1);
+
+        this.questionsSupport.assertEquals(questionFixtureEntity, questionFoundedEntity);
     }
 
     @Test
     public void testFindByTopicId() throws Exception {
-        final List<QuestionEntity> questions = this.questionsMapper.findByTopicId(2);
-        Assert.assertEquals((Integer) 1, (Integer) questions.size());
-        final QuestionEntity question = questions.get(0);
-
-        Assert.assertEquals((Integer) 3, question.getId());
-        Assert.assertEquals("test3", question.getText());
-        Assert.assertEquals((Integer) 3, question.getQuestionTypeId());
-        Assert.assertEquals((Integer) 2, question.getTopicId());
+        final List<QuestionEntity> questionsFixtureEntities = new ArrayList<>();
+        questionsFixtureEntities.add(this.questionsSupport.getEntityFixtureMock(0));
+        questionsFixtureEntities.add(this.questionsSupport.getEntityFixtureMock(1));
+        questionsFixtureEntities.add(this.questionsSupport.getEntityFixtureMock(2));
+
+        final List<QuestionEntity> questionsFoundedEntities = this.questionsMapper.findByTopicId(2);
+
+        Assert.assertEquals(1, questionsFoundedEntities.size());
+
+        Integer index = 2;
```

Comment:
Magic index 2 ? Correct test. Create  questionsFixtureEntities with appropriate items and iterate from index 0